src/lib/endpoints-to-oai31.helpers.ts

```diff
@@ -153,6 +146,8 @@ export const createPathParameterTypes = (
 
 export const createQueryParameterTypes = (
   queryParameters: Schema | undefined,
+  options: Options,
+  mostRecent?: any,
```

Comment:
This is still breaking `npm run lint`. Pretty sure this will fix it temporarily
```suggestion
  mostRecent?: unknown,
```

Reply:
`npm run lint` still fails with 2 issues - pretty sure changing `any`s to `unknown`s will fix it though :smile: